src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -276,6 +436,19 @@ IStreamsExecutor::Config IStreamsExecutor::Config::MakeDefaultMultiThreaded(cons
                 custom::info::default_concurrency(custom::task_arena::constraints{}.set_core_type(core_types.back()));
             num_cores_default = (num_big_cores_phys <= hyper_threading_threshold) ? num_big_cores : num_big_cores_phys;
         }
+        // if nstreams or nthreads are set, need to calculate the Hybrid aware parameters here
+        if (streamExecutorConfig._big_core_streams == 0) {
+            UpdateHybridCustomThreads(streamExecutorConfig,
+                                      num_big_cores_phys,
+                                      core_types.size() > 1 ? num_little_cores : 0);
```

Comment:
Why pass the 2nd and 3rd arguments? Which can be got inside of  UpdateHybridCustomThreads().